### PR TITLE
✨ Whats new pages

### PR DESCRIPTION
**Pull request by joshbermanssw**
We want pages on our site where can display the chanelog of both tinacms and tinacloud, this PR;
- [ ] creates 2 collections for whats new tinacms and whats new tinacloud 
- [ ] creates 2 pages for displaying the data inside the collection 

Screenshot 2024-08-14 at 3 40 59 PM
**Figure: Whats new with TinaCMS page**

### CODE REVIEW

File: components/layout/Footer.tsx [hunk 1]
```diff
@@ -58,6 +58,28 @@ const footerNav = [
         link: '/media',
         label: 'Media',
       },
+      // {
+      //   link: '/',
+      //   label: 'Releases',
+      // },
```
Comment (bradystroud) [inline components/layout/Footer.tsx:64]: should be removed if its not used

File: pages/whats-new/tinacms.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  h1: (props) => <h1 className="text-lg font-bold mt-8 mb-4" {...props} />,
+  h2: (props) => <h2 className="text-lg font-semibold mt-6 mb-4" {...props} />,
+  h3: (props) => <h3 className="text-lg font-semibold mt-5 mb-3" {...props} />,
+  h4: (props) => <h4 className="text-lg font-medium mt-4 mb-3" {...props} />,
+  h5: (props) => <h5 className="text-lg font-medium mt-3 mb-2" {...props} />,
+  h6: (props) => <h6 className="text-base font-medium mt-2 mb-2" {...props} />,
+  ul: (props) => <ul className="list-disc ml-5 mb-4" {...props} />,
+  ol: (props) => <ol className="list-decimal ml-5 mb-4" {...props} />,
+  li: (props) => <li className="mb-2" {...props} />
+};
```
Comment (bradystroud) [inline pages/whats-new/tinacms.tsx:19]: we are exporting this on both pages... can it be moved somewhere generic?

File: pages/whats-new/tinacloud.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+const Tinacloud = ({ items }) => {
+  console.log(items)
+  return (
+    <Layout>
+      <div className="p-6">
+        <div className="py-12 lg:py-16 last:pb-20 last:lg:pb-32 max-w-prose mx-auto">
+          <h3 className="text-center justify-center font-tuner text-3xl lg:text-4xl lg:leading-tight bg-gradient-to-br from-orange-400 via-orange-600 to-orange-700 group-hover:from-orange-300 group-hover:via-orange-500 group-hover:to-orange-700 bg-clip-text text-transparent">
+            What's new with TinaCloud
+          </h3>
```
Comment (bradystroud) [inline pages/whats-new/tinacloud.tsx:47]: This should be a h1

File: pages/whats-new/tinacms.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+const Tinacms = ({ items }) => {
+    console.log(items)
+  return (
+    <Layout>
+      <div className="p-6">
+        <div className="py-12 lg:py-16 last:pb-20 last:lg:pb-32 max-w-prose mx-auto">
+        <h3 className="text-center justify-center font-tuner text-3xl lg:text-4xl lg:leading-tight bg-gradient-to-br from-orange-400 via-orange-600 to-orange-700 group-hover:from-orange-300 group-hover:via-orange-500 group-hover:to-orange-700 bg-clip-text text-transparent">
+            What's new with TinaCMS
+          </h3>
```
Comment (bradystroud) [inline pages/whats-new/tinacms.tsx:43]: This should be a h1

File: pages/whats-new/tinacms.tsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+          <div className="mt-8">
+            {items.map((item) => (
+              <div key={item.id} className="mb-6 p-10 shadow-xl rounded-lg transform transition-transform duration-300 hover:scale-105">
+                <h4 className="text-2xl bg-gradient-to-br from-blue-700 to-blue-1000 bg-clip-text text-transparent text-blue-700 font-semibold">
+                  Version {item.versionNumber}
+                </h4>
```
Comment (bradystroud) [inline pages/whats-new/tinacms.tsx:49]: these should be h2s

File: pages/whats-new/tinacms.tsx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+                <p className="text-sm text-gray-500">
+                  Released on {new Date(item.dateReleased).toLocaleDateString()}
+                </p>
+                <TinaMarkdown content={item.body} components={versionComponents} />
+              </div>
+            ))}
+          </div>
+          <div className="font-tuner text-lg text-center text-blue-700">
+            <Link href="https://github.com/tinacms/tinacms/blob/main/packages/tinacms/CHANGELOG.md#patch-changes-4"> See More </Link>
```
Comment (bradystroud) [inline pages/whats-new/tinacms.tsx:58]:
```suggestion
            <Link href="https://github.com/tinacms/tinacms/blob/main/packages/tinacms/CHANGELOG.md"> See More </Link>
```

File: pages/whats-new/tinacloud.tsx
Comment (bradystroud) [inline pages/whats-new/tinacloud.tsx:1]: Looks bad without content

I'd add a 'no release yet' or something to show its not broken just empty

image

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: components/layout/Footer.tsx (hunk 1 above)
Comment (joshbermanssw) [reply components/layout/Footer.tsx]: ✅

File: pages/whats-new/tinacms.tsx (hunk 2 above)
Comment (joshbermanssw) [reply pages/whats-new/tinacms.tsx:19]: Done, moved to component/styles/WhatsNewVisualComponents.tsx

File: pages/whats-new/tinacloud.tsx
Comment (joshbermanssw) [reply pages/whats-new/tinacloud.tsx:1]: @wicksipedia is planning on inputting content

File: pages/whats-new/tinacloud.tsx (hunk 3 above)
Comment (joshbermanssw) [reply pages/whats-new/tinacloud.tsx]: done

File: pages/whats-new/tinacms.tsx (hunk 5 above)
Comment (joshbermanssw) [reply pages/whats-new/tinacms.tsx]: done

File: pages/whats-new/tinacloud.tsx
Comment (joshbermanssw) [reply pages/whats-new/tinacloud.tsx:1]: as per conversation added null condition

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: pages/whats-new/tinacloud.tsx [hunk 7]
```diff
@@ -0,0 +1,59 @@
+import { Layout } from 'components/layout'
+import client from 'tina/__generated__/client'
+import { TinaMarkdown } from 'tinacms/dist/rich-text'
+import { versionComponents } from 'components/styles/WhatsNewVisualComponents'
```
Comment (bradystroud) [inline pages/whats-new/tinacloud.tsx:4]: Confusing naming - what are version components?
```suggestion
import { visualComponents } from 'components/styles/WhatsNewVisualComponents'
```

File: pages/whats-new/tinacms.tsx [hunk 8]
```diff
@@ -0,0 +1,64 @@
+import { Layout } from 'components/layout'
+import client from 'tina/__generated__/client'
+import { TinaMarkdown } from 'tinacms/dist/rich-text'
+import { FaPlus, FaMinus } from 'react-icons/fa'
+import { useState } from 'react'
+import Image from 'next/image'
+import Link from 'next/link'
+import { versionComponents } from 'components/styles/WhatsNewVisualComponents'
```
Comment (bradystroud) [inline pages/whats-new/tinacms.tsx:8]: same as above

File: components/styles/WhatsNewVisualComponents.tsx
Comment (bradystroud) [inline components/styles/WhatsNewVisualComponents.tsx:1]: Confused why we have separate heading components for the whats new pages.

Headings should be consistent across the site

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: tina/schema.tsx [hunk 9]
```diff
@@ -619,5 +619,27 @@ export const schema = defineSchema({
...
+      name: 'WhatsNewTinaCloud',
+      label: 'Whats new - TinaCloud',
+      path: 'content/whats-new-tinacloud',
+      format: 'mdx',
+      fields: [
+        { name: 'versionNumber', label: 'Version Number', type: 'string' },
+        { name: 'dateReleased', label: 'Date Released', type: 'datetime'},
+        { name: 'body', label: 'Body', type: 'rich-text', isBody: true, description: 'The content of the release notes. Note that h1-h5 are the same size (i.e text-lg in tailwind).' },
+      ],
+    }
```
Comment (bradystroud) [inline tina/schema.tsx:643]: Since the fields are duplicated, can they be moved to a component and referenced here?

**Review by bradystroud — APPROVED** [review]
LGTM

**Pull request merged**